module/importer/src/Infrastructure/Handler/ImportSimpleProductCommandHandler.php

```diff
@@ -48,13 +59,24 @@ public function __invoke(ImportSimpleProductCommand $command): void
                 }
                 $categories[] = new CategoryCode($category);
             }
-
-            $product = $this->action->action(
-                new Sku($command->getSku()),
+            $attributesToRedispatch = $this->attributeValidatorImportAction->action($command->getAttributes());
+            $validatedAttributes = array_diff_key($command->getAttributes(), $attributesToRedispatch);
+            $sku = new Sku($command->getSku());
```

Comment:
You shoulden't  convert string to sku object without validation, SKU value exists in given command can be invalid.  
Validation for this is handler in action class.